lib/audit_help.c

```diff
@@ -44,6 +45,23 @@ void audit_help_open (void)
 	}
 }
 
+/*
+ * This takes a string and replaces the old character with the new.
+ */
+static inline char *strtr (char *str, char old, char new)
```

Comment:
Please put the type on a separate line.

Also, please don't put a space before the opening parentheses (some old code still has it, but we're getting rid of that in new code).

```c
static inline char *
strtr(char *str, char old, char new)
```

Reply:
I am following the style that I see already existing in that file. Never-the-less, I can fix this.